app/test_main.py

```diff
@@ -1,3 +1,63 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected",
+    [
+        pytest.param(
+            0,
+            0,
+            [0, 0],
+            id="cat and dog ages equal zero"
+        ),
+        pytest.param(
+            14,
+            14,
+            [0, 0],
```

Comment:
The expected output for 14 cat/dog years should not be [0, 0] if the conversion logic is consistent with the other test cases. Please verify the expected output for this case.